apps/communications/dialer/test/marionette/lib/views/activities/views.js

```diff
@@ -19,6 +19,18 @@ ActivitiesView.prototype = {
       var contacts = this.client.loader.getAppClass('contacts');
       contacts.switchToCreateNewContactActivity();
       return contacts;
+  },
+
+  pickAddToContact: function(){
+      this.client.waitFor(function() {
+        return this.accessors.addToContactMenuItem.enabled();
```

Comment:
I just realized this wait should be in the accessors. Accessors make sure an HTML element is ready to be used by the view. Sorry for missing that in the previous review.